lib/mindwendel_web/controllers/static_page_html/home.html.heex

```diff
@@ -63,6 +69,11 @@
     <% end %>
   </main>
   <footer class="mt-auto">
-    <p>Made with &hearts; by Gerardo, Jannik and Nicho</p>
+    <span class="me-2">Made with ❤️ in the 🇪🇺</span>
+    <%= if Application.fetch_env!(:mindwendel, :options)[:feature_privacy_imprint_enabled] do %>
```

Comment:
Having a module that wraps feature flags would avoid leaking the path/keys to the feature flag throughout the application :)